types/html-to-text/index.d.ts

```diff
@@ -292,6 +248,11 @@ export interface FormatOptions {
      * @default ['[', ']']
      */
     linkBrackets?: [string, string] | false | undefined;
+    /**
+     * A function to rewrite link href attributes and image src attributes.
+     * Applied before baseUrl.
+     */
+    pathRewrite?: ((path: string, meta: metaData) => string) | undefined;
```

Comment:
This might need "Only for" comment for consistency with other doc strings. (Less important than the other one, since the wording here is pretty clear by itself.)

It is only for specific formatters: https://github.com/html-to-text/node-html-to-text/blob/5aa6b573b940c35173f2d4c38ce652ecf98851a8/packages/html-to-text/README.md?plain=1#L227

Reply:
Updated.